app/src/org/storymaker/app/CacheWordActivity.java

```diff
@@ -170,6 +175,15 @@ public void onClick(View v) {
                         PassphraseSecrets secrets = PassphraseSecrets.fetchSecrets(CacheWordActivity.this, defaultPin);
                         mCacheWordHandler.changePassphrase(secrets, mTextCreatePin.getText().toString().toCharArray());
                         Timber.d("replaced default pin");
+
+                        // moving this code here to prevent accidental locks
+
+                        // set status to prevent use of default pin
+                        SharedPreferences sp = getSharedPreferences("appPrefs", Context.MODE_PRIVATE);
+                        SharedPreferences.Editor e = sp.edit();
+                        e.putString("cacheword_status", BaseActivity.CACHEWORD_SET);
+                        e.commit();
```

Comment:
this is the value everything looks at to see whether it should automatically enter the default pin or prompt the user so i'm inclined to leave it as-is rather than risk another unexpected state.